java-checks/src/test/files/checks/CompareToNotOverloadedCheck.java

```diff
@@ -45,4 +45,5 @@ int compareTo(A a) {      // Compliant
       return 1;
     }
   }
+  
 }
```

Comment:
Nop. I meant that the file should end with an empty line, not with `}` (see the small icon above?)

Reply:
But it's ok, we are gonna live with it.